README.md

```diff
@@ -1516,6 +1517,132 @@ FunctionDeclaration(path) {
 
 ----
 
+# Building Nodes
+
+When writing transformations you'll often want to build up some nodes to insert
+into the AST. As mentioned previously, you can do this using the
+[builder](#builder) methods in the [`babel-types`](#babel-types) package.
+
+The method signature for a builder is simple, the name is the same as the node
+type you want to build, but with the first letter lowercase. For example if you
```

Comment:
`The method name for a builder is simply the name of the node type you want to build except with the first letter lowercased.`